.github/workflows/timeseries-run-automation-tests.yaml

```diff
@@ -0,0 +1,49 @@
+name: "[Time Series Analytics] Run weekly automation tests"
+run-name: "[Time Series Analytics] Run weekly automation tests"
+on:
+  schedule:
+    - cron: '0 14 * * 5' # 14:00 UTC
+  workflow_dispatch:
+permissions: {}
+
+jobs:
+  time-series-analytics-microservice-weekly:
+    name: Weekly-Run TimeSeries Analytics Microservice tests
```

Comment:
Let's update the existing PR workflow to add in a step to run just the happy path test cases execution (health check, docker and helm deployment w/ logs verification).